### PR TITLE
Feature: implement-manufacturer-registration form

### PR DESCRIPTION
**Pull request by KevinLatino**
Feature: implement-manufacturer-registration

Closes: #34 

image

### CODE REVIEW

**Review by zintarh — COMMENTED** [review]
Please add background image to the page as discussed

File: frontend/src/app/components/ManufacturerForm.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                placeholder="Enter company name"
+              />
+            </div>
+          </div>
+
+          <div>
+            <label htmlFor="manufacturer-name" className="block text-sm font-medium text-gray-200 mb-2">
+              Manufacturer Name
+            </label>
+            <div className="relative rounded-md shadow-sm">
```
Comment (zintarh) [inline frontend/src/app/components/ManufacturerForm.tsx:47]: Also please make the input fields much bigger

**Review by zintarh — COMMENTED** [review]
Please fix the CI too

**Comment by KevinLatino** [comment]
image

**Comment by zintarh** [comment]
@KevinLatino Please can you fix CI

**Comment by zintarh** [comment]
Lgtm

**Review by zintarh — APPROVED** [review]

**Pull request merged**